test/unit/session.spec.js

```diff
@@ -31,6 +98,16 @@ describe('util/session', () => {
           .respondWithData(() => testData.extendedUsers.first());
       });
 
+      it('saves currentUser', () => {
```

Comment:
This test is a little unrelated, but I noticed it was missing while working on the tests above.